packages/data/src/registry.js

```diff
@@ -221,6 +263,16 @@ export function createRegistry( storeConfigs = {}, parent = null ) {
 		dispatch,
 		use,
 		register,
+		getStoreAtom,
```

Comment:
Note to myself: check why `getStoreAtom` rather than `getAtomStore` and as a consequence `atomStores`.

Reply:
We're creating an "atom" for the "store" :) (so it's not a store of atoms)

Reply:
`getAtomForStore` - would it work better or does it read the same?

The name `atomsForStores` doesn't sound better though 😅